help/deploy/aws/cloudformation/templates/marketplace.template.yml

```diff
@@ -13,10 +13,10 @@ Mappings:
       repo: "datagrok/grok_spawner"
     JKG:
       domain: "709825985650.dkr.ecr.us-east-1.amazonaws.com"
-      repo: "datagrok/jupyter_kernel_gateway"
+      repo: "datagrok/jupyter"
     JN:
       domain: "709825985650.dkr.ecr.us-east-1.amazonaws.com"
-      repo: "datagrok/jupyter_notebook"
+      repo: "datagrok/jupyter"
```

Comment:
Why do you use two entries instead of one as in basic?

Reply:
I don't actually understand where Mappings.Images.JKG(or JN) will be used, so left both

Reply:
But you changed it in Basic, they are used the same way